test/HardwareObjectsMockup.xml/mxcube-web/ui.yaml

```diff
@@ -155,3 +155,67 @@ camera_setup:
       url: http://localhost:9090/mjpg/video.mjpg?streamprofile=mxcube
       width: 800
       height: 450
+
+plate_setup:
```

Comment:
Super nice idea ! :1st_place_medal:. It should perhaps however be configured on hardware object level ? Its really nice with this configuration but I feel that its less a UI configuration and more specific to the hardware object ?

Reply:
I closed this one i'll create a new one with just  the css  changes